components/menu/stories/menu.stories.js

```diff
@@ -68,310 +97,189 @@ export default {
 			}
 		},
 	},
-};
-
-export const Default = Template.bind({});
-Default.storyName = "Standard with Dividers";
-Default.args = {
-	items: [
-		{ label: "Deselect" },
-		{ label: "Select inverse" },
-		{ label: "Feather" },
-		{ label: "Select and mask" },
-		{ type: "divider" },
-		{ label: "Save selection" },
-		{ label: "Make work path", isDisabled: true },
-	],
-};
-
-export const Truncate = Template.bind({});
-Truncate.args = {
-	items: [
-		{ label: "Deselect", },
-		{ label: "Select inverse" },
-		{ label: "Feather" },
-		{ label: "Select and mask" },
-		{ type: "divider" },
-		{ label: "Save selection" },
-		{ label: "Make work path", isDisabled: true },
+	decorators: [
+		(Story, context) => {
+			if (!window.isChromatic()) return Story(context);
```

Comment:
When I first looked at the Menu with Variants, specifically with dividers off, I didn't realize that the bold text were headers. I wonder if we should include this styling or some variation of it to help separate them even outside of the testing preview

![Screen Shot 2024-05-15 at 11 30 21 AM](https://github.com/adobe/spectrum-css/assets/23404383/fa27715d-3586-4a90-9762-34ee22806767)

Reply:
Good point! I tried to explicitly give the headers header-like names and used that seafoam color to color the testing preview heading to separate them from the rest of the component, since we weren't using that anywhere. Open to other ideas and discussion on this.

<img width="300" alt="image" src="https://github.com/adobe/spectrum-css/assets/54716846/f94e26a0-e4ee-4abb-a48a-2702d2afca64">